internal/database/postgres/migration/000005_test-database-create.up.sql

```diff
@@ -0,0 +1 @@
+create database postgrestest;
```

Comment:
Ah... here is it :D 
I saw a couple of other PRs, where it was missing

Reply:
Make sure that order of migrations is correct, because sequence 5 is used in #45 for table insert